research/README.md

```diff
@@ -1,6 +1,10 @@
 # Open UI
 
-## Getting started
+The purpose of [Open UI](https://open-ui.org/) to the web platform is to allow web developers to style and extend built-in web UI controls, such as <select> dropdowns, checkboxes, radio buttons, and date/color pickers.
```

Comment:
You should put the quote ticks around <select>

Reply:
Good spot! I added some blockquotes around it